extension/scripts/features/hidegymhighlight/ttHideGymHighlight.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Hide Gym Highlight",
+		"sidebar",
+		() => settings.pages.sidebar.hideGymHighlight,
+		null,
+		hideGymHighlight,
+		removeHiddenHighlight,
+		{
+			storage: ["settings.pages.sidebar.hideGymHighlight"],
+		},
+		async () => {
+			await requireSidebar();
+		}
+	);
+
+	function hideGymHighlight() {
+		const navGym = document.find("#nav-gym");
+		const gymClass = [...navGym.classList].find((name) => name.includes("available___"));
```

Comment:
`navGym.classList.contains("^=available___")` is better 
It's a custom edit of the `contains` function.

Reply:
Added.